lib/src/binary_codec.dart

```diff
@@ -292,6 +411,26 @@ class PostgresBinaryDecoder extends Converter<Uint8List, dynamic> {
     }
   }
 
+  List<T> readListBytes<T>(
```

Comment:
Similar to the other point: `package:buffer`'s reader should be used here.

Reply:
Done